src/object/schema_class_truncator.cpp

```diff
@@ -595,7 +595,7 @@ namespace cubschema
 	    return error;
 	  }
 
-	cnt_refers = db_get_int (&value);
+	cnt_refers = db_get_bigint (&value);
```

Comment:
Please check the type of return value.

Reply:
After consideration of it, you can merge this PR whether or not confirm by me.

Reply:
Thank you, I changed it from `int` to `DB_BIGINT`